ui/dgt/src/play.ts

```diff
@@ -96,6 +96,7 @@ export default function (token: string) {
   let liveChessConnection: WebSocket; //Connection Object to LiveChess through websocket
   let isLiveChessConnected = false; //Used to track if a board there is a connection to DGT Live Chess
   let currentSerialnr = '0'; //Public property to store the current serial number of the DGT Board in case there is more than one
+  let currentBoard: { serialnr: string; state: string }; //Stores active board object
```

Comment:
why do we have both `currentSerialnr` and `currentBoard.serialnr`?

Reply:
currentSerialnr is used in the DGTliveChessConnectionLoop that checks whether it's been assigned a non-null value to determine connection status to the board and in the socket connection's onclose to reset the value back to 0. In line 859 there's a conditional checking a hard coded values from the first boards array index value. 

It seems like the code was initially designed to use both a variable to hold the serial number and then it also checks hardcoded array values(assuming the user only has one board) in the code.

Reply:
I think this was mostly a rhetorical question. Is there a reason why we **need** to have both? Because otherwise, it should be changed so that only one value is used everywhere.

Reply:
Yeah, true. Could simply just use currentBoard. Why do we need to pull up two tabs in order for this code to work?

Reply:
Got it. I'll change it. I was also thinking about making this work with the regular interface so you don't need two tabs open to use this feature. 

Sorry for the time between responses, I've been busy with work.